Assets/Code/Scripts/Level generation scripts/TavernGenerationScript.cs

```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+
+public class TavernGenerationScript : TileGenerationScript
+{
+    public override GameObject Generate(int i, int j)
+    {
+        //we have the tavern tiles.
+        if (i % (Size - 1) == 1)
+        {
+            return Instantiate(tiles[0], new Vector3(i * Width, 0, j * Width), 
+                Quaternion.identity);
+        }
+        
+        if (j % (Size - 1) == 1)
+        {
+            return Instantiate(tiles[0], new Vector3(i * Width, 0, j * Width),
+                Quaternion.Euler(0, 270, 0));
+        }
+        
+        if (i % (Size - 1) == Size - 2)
+        {
+            return Instantiate(tiles[0], new Vector3(i * Width, 0, j * Width),
+                Quaternion.Euler(0, 180, 0));
+        }
```

Comment:
Would it be possible for some of this to be in an `else if` format instead?

Reply:
That's what I originally had, but rider was saying that since it is an early return statement, which you only use if. 
Normally you use else if if the code is in the middle of a function. If you are at the beginning and are checking for an exit status it is easier for people to read if it's just if statements